### PR TITLE
Backend changes for pod actions

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: casepro/pods/views.py [hunk 1]
```diff
@@ -33,4 +37,15 @@ def perform_pod_action(request, pk):
         return JsonResponse(
             {'reason': 'JSON decode error', 'details': e.message}, status=400)
 
-    return JsonResponse(pod.perform_action(data))
+    case_id = data.get('case_id')
+    if case_id is None:
+        return JsonResponse(
+            {'reason': 'Request object needs to have a "case" field'}, status=400)
```
Comment (justinvdm) [inline casepro/pods/views.py]: s/`case`/case_id`, I think

**Comment by justinvdm** [comment]
One minor comment, looks good :)

File: casepro/pods/views.py (hunk 1 above)
Comment (rudigiesler) [inline casepro/pods/views.py]: Yes, I forgot to change the error message when I changed the field above, thanks.

Resolved in https://github.com/praekelt/casepro/pull/51/commits/5ab83af30e128619399b2f7917265d23c1565b1d

**Comment by rudigiesler** [comment]
So apparently merging in the latest from `EPIC-OPENHELPD-23` broke this PR, and I'm not sure how to fix it.. (If I diff between this branch and `EPIC-OPENHELPD-23` on my local machine, it looks okay)

**Comment by rudigiesler** [comment]
Succeeded by https://github.com/praekelt/casepro/pull/55

**Pull request closed without merging**